xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/Translations.xml

```diff
@@ -78,7 +78,7 @@ notifications.filters.preferences.allEvents=-
 notifications.filters.preferences.addPreference.filterType.inclusive=Inclusive
 notifications.filters.preferences.addPreference.filterType.exclusive=Exclusive
 notifications.filters.preferences.addPreference.notificationFormat.label=Notification format
-notifications.filters.preferences.addPreference.notificationFormat.hint=On which format of notification the filter should apply?
+notifications.filters.preferences.addPreference.notificationFormat.hint=On which format of notification by the filter should apply?
```

Comment:
I don't think there was an error here. In fact I don't understand the new version.